test/model.js

```diff
@@ -163,7 +163,7 @@
     assert.ok(!a.isNew(), "any defined ID is legal, negative or positive");
     a = new Backbone.Model({'foo': 1, 'bar': 2, 'baz': 3, 'id': 0});
     assert.ok(!a.isNew(), "any defined ID is legal, including zero");
-    assert.ok( new Backbone.Model({          }).isNew(), "is true when there is no id");
```

Comment:
This is separate from the rest (but a possible good use of this rule...) I'll make the change separately now.